frontend/src/container/LogDetailedView/LogDetailedView.types.ts

```diff
@@ -23,3 +23,7 @@ export interface IFieldAttributes {
 export interface JSONViewProps {
 	logData: ILog;
 }
+
+export interface InfraMetricsSProps {
+	logData: ILog;
+}
```

Comment:
these two interfaces seem the same, should we define InfraMetricsProps as a type alias to JSONViewProps (i.e. type InfraMetricsProps = JSONViewProps?